cms/articles/models.py

```diff
@@ -474,3 +483,39 @@ def get_context(self, request: "HttpRequest", *args: Any, **kwargs: Any) -> dict
         context["has_notices"] = bool(notices)
 
         return context
+
+    def ld_entity(self) -> dict[str, object]:
+        """Add statistical article specific schema properties to JSON LD."""
+        # TODO pass through request to this, once wagtailschemaorg supports it
+        properties = {
+            "url": self.get_full_url(),
+            "headline": self.seo_title or self.listing_title or self.title,
+            "description": self.search_description or self.listing_summary or strip_tags(self.summary),
+            "datePublished": self.release_date.isoformat(),
```

Comment:
Worth checking, though schema.org suggests this is Date or DateTime, our requirements doc suggests `DateTime`.

Reply:
Good point, it seems Google prefer a DateTime here too, though schema.org allows either. The page release date appears to only be a date, so would the publish date time be more appropriate?

Reply:
See https://github.com/ONSdigital/dis-wagtail/pull/250#discussion_r2201873669